stylesheets/styles.css

```diff
@@ -0,0 +1,363 @@
+   
+  header {
+    height: 53.8rem;
+    background: url(../assets/images/hero-img.jpg);
+    background-repeat: no-repeat;
+    background-position: center -35rem;
+    background-size: cover; 
+    padding-top: 17rem;
+  }
+  
+  header:hover{
+    cursor: pointer;
+  }
+  
+  header h1{
+    background: url(../assets/images/h1.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+    background-size: auto 10rem;
+    margin-bottom: 1rem;
+  }
+  
+  header h2{
+    background: url(../assets/images/h2.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+    margin-bottom: 1.2rem;
+  }
+  
+  header h3{
+    background: url(../assets/images/cta-simple.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+  }
+  
+  header h3:hover{
+    background: url(../assets/images/cta-hover.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 7rem;
+    cursor: pointer;
+  }
+  
+  aside {
+    display: flex;
+    height: 20.6rem;
+    border-top: 2px solid white;
+  }
+
+  aside > div {
+    flex: 1;
+  }
+  
+  aside div:hover {
+    cursor: pointer;
+  }
+  
+  .product{
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover; 
+    height: 100%;
+    width: 100%;
+    display: block;
+  }
+  
+  .product-gallery div:not(:last-child){
+    border-right: 2px solid white;
+  }
+  
+  .a-1{
+    background-image: url(../assets/images/product-1.jpg);
+  }
+  
+  
+  .a-2 {
+    background-image: url(../assets/images/product-2.jpg);
+  }
+  
+  .a-3 { 
+    background-image: url(../assets/images/product-3.jpg);
+  }
+  
+  .a-4 {
+    background-image: url(../assets/images/product-4.jpg);
+  }
+  
+  footer {
+    /* display: flex; */
+    /* padding: 2.5rem 10rem; */
+  }
+  
+  footer > div {
+    flex: 1;
+  }
+  
+  
+  .links-column-1{
+    max-width: 60.5rem;
+  }
+  
+  .links-column-2{
+    margin-left: 3rem;
+  }
+  
+  
+  footer {
+    color: #908e8e;
+    font-size: 1.1rem;
+  }
+  
+  footer ul{
+    list-style: none;
+    display: flex;
+  }
+  
+  footer a {
+    color: rgb(72,169,218);
+    text-decoration: none;
+  }
+  
+  .links-column-2 ul li a {
+    padding: 0 .6rem;
+  }
+  
+  .links-column-2 ul li:not(:last-child) a{
+    border-right: 1px solid  rgb(204,204,204);
+  
+  }
+  
+  
+  .links-column-1 div:first-child{
+    padding-bottom: 1rem;
+  }
+  
+  .links-column-1 {
+    padding-left: 10rem;
+  } 
+  
+  a:hover{
+    text-decoration: underline;
+  }
+  
+  @media screen and (max-width: 1350px) {
+    footer, .links-column-2 ul {
+      flex-direction: column;
+      justify-content: center;
+      text-align: center;
+    }
+  
+    .links-column-1{
+      padding-left: 0;
+      max-width: 100%;
+    }
+  
+    .links-column-2 {
+      margin-top: 2.9rem;
+    }
+  
+    .links-column-2 ul li a {
+      border: none !important;
```

Comment:
#### `REQUIRED`

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`.

Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

> Make sure you DON’T make [common mistakes](https://speckyboy.com/good-bad-css-practices/) in your CSS styles.

> **Recommendation:** You should only use the `!important` rule when overriding styles in a user style sheet, overriding 3rd party code and inline styles as well as utility classes.